eng/PoliCheckExclusions.xml

```diff
@@ -0,0 +1,3 @@
+<PoliCheckExclusions>
```

Comment:
is this still where exclusions go, in the new world? I assume not and we don't need this file. It's probably json somewhere

Reply:
I don't think so. My guess is that it's a config setting in the yaml.

Reply:
Okay let's keep the file, move to .config, and add the exclusion file setting under the policheck yaml bit

exclusionsFile: $(Build.SourcesDirectory)\.config\PoliCheckExclusions.xml

e.g.

```
policheck:
  enabled: true
  exclusionsFile: $(Build.SourcesDirectory)\.config\PoliCheckExclusions.xml
```

Reply:
done